src/components/calcite-panel/calcite-panel.tsx

```diff
@@ -54,6 +54,11 @@ export class CalcitePanel {
    */
   @Prop({ reflect: true }) dismissible = false;
 
+  /**
+   * Specifies the maxiumum height of the panel.
+   */
+  @Prop({ reflect: true }) heightScale: "s" | "m" | "l";
```

Comment:
Would we add separate `scale` props to control other parts of the panel if that became desired in the future? If the intent would be to have one `scale` per component, I'd drop the height part since it may expand beyond that one day.

Reply:
> Would we add separate scale props to control other parts of the panel if that became desired in the future

Potentially. This only affects panel height so we can either say that in both the property name and the readme description or just the readme description.

Reply:
The property having a name specific to the height is probably good since it clearly excludes width.

This is one of the places where we probably want to treat the width like we treat the width of the shell-panels, e.g. the developer shouldn't need to worry too much about it.